### DISCUSSION TITLE
Auth Attempt Access Denied

### DISCUSSION THREAD

**Discussion by yaz231** [UltimaHoarder/UltimaScraper#1796] [Q&A]
I searched through the Network Debugger in Microsoft Edge, I added in all the components as shown in the README page. However, I keep getting my access denied and I am not sure why. I have tried web browser authentication, as well as through the cookie. Not really sure what I am doing wrong and would love some help on this one.

Auth (V1) Attempt 10/10
Access denied.

**Comment by adrianoadum** [comment]
Same issue here. I configured the cookie and user_agent accordingly, but when the script tries to authenticate looks the session is invalidated by OF, because I need to login again in the browser.

**Comment by fightcancer2** [comment]
Does your auth.json look like this:

yyyyyyy = your auth id which is repeated thrice below
xxxxxxxxxxxxxxxxxxxxx = your session ID
zzzzzzzzzzzzzzzzzzzzz = your x-bc header value

{
  "auth": {
    "username": "default",
    "cookie": "auth_id=yyyyyyy; sess=xxxxxxxxxxxxxxxxxx; auth_hash=; auth_uniq_yyyyyyy=; auth_uid_yyyyyyy=;",
    "x_bc": "zzzzzzzzzzzzzzz",
    "user_agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:95.0) Gecko/20100101 Firefox/95.0",
    "email": "",
    "password": "",
    "hashed": false,
    "support_2fa": true,
    "active": true
  }
}

**Reply by yaz231** [reply]
I don't have auth_unique_yyyyyyy or  auth_uid_yyyyyyy; otherwise, yes.

**Comment by yaz231** [comment]
@fightcancer2, I still haven't been able to piece through this. I'll figure it out eventually, do you mind deleting this discussion? This Github account isn't a burner and I'd like to keep it off these kinds of repos lol
